collections/ansible_collections/osac/templates/README.md

```
@@ -223,16 +224,18 @@ When implemented, `hcp_data_plane` will support provisioning multiple StorageCla
 into the guest HCP cluster (e.g., separate tiers for databases and general workloads).
 Currently, all CaaS targets return an explicit "not yet implemented" error.
 
-**Configuration:** Storage tiers are configured via the `STORAGE_TIERS` env var in the
-`storage-operations-ig` ConfigMap:
+**Configuration:** Storage tiers arrive via the `ansible_eda.event.storage_tier_definitions`
+extra_var, populated by osac-operator from the Tier API:
 ```json
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)